src/main/java/intellistart/interviewplanning/controllers/dto/CandidateDto.kt

```diff
@@ -0,0 +1,10 @@
+package intellistart.interviewplanning.controllers.dto
+
+/**
+ * DTO for user with CANDIDATE role.
+ */
+data class CandidateDto (
+    val email: String
+) {
+    val role = "CANDIDATE"
```

Comment:
looks like useless parameter, but I see you had it before

Reply:
Any unregistered user in the system is a candidate (a user whose e-mail is not stored in the system's database, and the e-mail in turn is the mail that is linked to the user's Facebook account). That is, only interviewers and coordinators are stored in the system database. My API provides a response to mapping "GET /me" to the fields "email" and "role". So I consider the field "val role = "CANDIDATE"" in CandidateDto be the right solution when the user is a candidate.

Reply:
My point is that you can understand that it is a candidate from class, and the role seems like not needed
but it's minor issue, you can leave it as is